tools/build-release-branch.sh

```diff
@@ -52,6 +52,12 @@ function create_release_gitignore {
 
     # Add custom stuff to .gitignore release
     echo "/_inc/client" >> .gitignore
+    # Needs to stay in sync with .svnignore
+    # The quoting is important here so the shell expands the contents producing multiple strings.
+    # .gitignore will not respect the {brace,variations} so we'll produce multiple lines.
+    #
+    #                       ↓ this brace is unquoted. Important!
+    printf "%s\n" "/extensions/**/*."{"js","jsx","json","gif","png","css","jpg","jpeg","sass","scss","svg"} >> .gitignore
 
     # Remove old .gitignore
     rm .gitignore-tmp
```

Comment:
Was about to comment about spaces as indentation on your new lines but realised this file is already a mixture of tabs/spaces.